src/main/java/org/lsc/Hooks.java

```diff
@@ -63,31 +64,56 @@ public class Hooks {
 	 *
 	 * return nothing
 	 */
-	public final static void postSyncHook(final String hook, final LscModifications lm) {
+	public final static void postSyncHook(final String hook, final String outputFormat, final LscModifications lm) {
```

Comment:
style: in general in Java we don't use static methods. You can construct the object in the constructor of the caller.